adapters/postgres/postgres.go

```diff
@@ -200,14 +196,7 @@ func CountByRequest(req *http.Request) (countQuery string) {
 
 // Query process queries
 func Query(SQL string, params ...interface{}) (jsonData []byte, err error) {
-	validQuery := chkInvalidIdentifier(SQL)
```

Comment:
now i see, so you have to add the checker on order by if not has yet, sorry for that